tests/test_flow/test_persistence_aip.py

```diff
@@ -107,3 +107,22 @@ def x_plus_one(x):
         r"Submitting AI Platform task .*\(name='x_plus_one'\).*CaseKey\(x=1\).*",
         r".*error while doing remote computation for x_plus_one\(x=1\).*AipError.*",
     )
+
+
+# This test may not work if the current Bionic git branch is not pushed to the
+# main github repo.
+@pytest.mark.real_gcp_only
+@pytest.mark.no_parallel
+def test_aip_with_docker_build(aip_builder):
+    builder = aip_builder
+    builder.set("core__aip_execution__docker_build_enabled", True)
+    builder.set("core__aip_execution__docker_image_name", None)
+
+    @builder
+    @bn.aip_task_config("n1-standard-4")
+    def x():
+        return 1
```

Comment:
Is there anything we can do here to validate that the environment we're running in matches the local one?

Reply:
Good suggestion. Maybe we can run a `pip freeze` and verify that the output is the same as the local environment.

Reply:
This entity can return the output of `pip freeze`. Pip entry for bionic is excluded because it does not have access to github.